action/GlobalPlanner.action

```diff
@@ -0,0 +1,9 @@
+# Motion planning request to the global planner
+string planning_group
+moveit_msgs/MotionSequenceRequest desired_motion_sequence
```

Comment:
I'm not really convinced of this name. Simply `request` would be fine imo

Reply:
I think calling this motion_sequence (As it is in the HybridPlanner.action) is actually clearer when used in code as it is a sub-message of the action goal.